docs/userguides/developing_plugins.md

```diff
@@ -103,6 +103,18 @@ entry_points={
 ...
 ```
 
+If you're using `pyproject.toml` you can register the entry_point like this
+
+`pyproject.toml`:
+
+```yml
+[...]
+# Make it work with cli like ape myplugin
+[project.entry-points.ape_cli_subcommands]
```

Comment:
I needed to register the entry-point using `[<project-name>.entry-points.ape_cli_subcommands]` in order for it to register properly.

Reply:
That's strange. I believe it has to be a property of the project. What env management tool are you using? Have you tried installing it using pip?

Reply:
I installed using `pip install -e .`. It actually gave me an error when I tried using what you have.
```
ValueError: invalid pyproject.toml config: `project`.
      configuration error: `project` must contain ['name'] properties
```

Reply:
Hmm that's odd

Reply:
Actually I think you need to define the project name like this 

```yml
[project]
name = "ape-utils"
[...]

[project.entry-points.ape_cli_subcommands]

[...]
```

Reply:
We'll probably need to add a new documentation section for using only a `pyproject.toml` for an ape project since it'll need more configuration than just this. Definitely something to keep in mind when we start development on this later @bitwise-constructs